core/mocks/trezortranslate_keys.pyi

```diff
@@ -253,7 +253,6 @@ class TR:
     device_name__change_template: str = "Change device name to {0}?"
     device_name__title: str = "Device name"
     entropy__send: str = "Do you really want to send entropy?"
-    entropy__title: str = "Internal entropy"
     entropy__title_confirm: str = "Confirm entropy"
```

Comment:
Is the `entropy__title_confirm` still used somewhere?

Reply:
Yes, it's a different thing. When you use `trezorctl crypto get-entropy`.